addons/hr_work_entry/security/hr_work_entry_security.xml

```diff
@@ -12,4 +12,10 @@
             <field name="perm_read" eval="0"/>
         </record>
 
+        <record id="ir_rule_hr_work_entry_type_multi_company" model="ir.rule">
+            <field name="name">HR Work Entry: Multi Company</field>
+            <field name="model_id" ref="model_hr_work_entry_type"/>
+            <field name="domain_force">['|', ('country_id', '=', False), ('country_id', 'in', user.env.companies.mapped('country_id').ids)]</field>
```

Comment:
```suggestion
        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

Reply:
work entry type model does not have company_id field and according to specs, differentiation of work entry types is based on countries rather than companies.